spec/models/offspring_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+include Warden::Test::Helpers
+
+RSpec.describe Offspring, type: :model do
+  active_record_offspring = 'activerecord.errors.models.offspring.attributes'
+
+  after(:each) do
+    Warden.test_reset!
+  end
+  describe "validating attribute first_name " do
+    it "is invalid without first_name" do
+      off = FactoryGirl.build(:offspring, first_name: "")
+      off.valid?
+      expect(off.errors[:first_name]).to include(I18n.t('f_name.blank', scope: active_record_offspring))
```

Comment:
f_name is a new translation I created in the en.yml file because otherwise it would not pass Rubocop because the line was to long. So i created a shorter version of first_name.blank -> f_name.blank